src/views/PositionPlot/PositionPlotView.tsx

```diff
@@ -52,35 +53,58 @@ const PositionPlotView: FunctionComponent<Props> = ({data, width, height}) => {
         context.stroke()
         context.setLineDash([]);
 
-        for (let dim of props.dimensions) {
-            context.strokeStyle = colorForUnitId(dim.dimensionIndex)
-            context.beginPath()
-            let first = true
-            for (let i=0; i<dim.pixelTimes.length; i++) {
-                const x = dim.pixelTimes[i]
-                const y = dim.pixelValues[i]
-                if (first) context.moveTo(x, y)
-                else context.lineTo(x, y)
-                first = false
-            }
-            context.stroke()
+        if (props.plotType === 'line' || !props.plotType) {
+            props.dimensions.forEach(dim => {
+                context.strokeStyle = colorForUnitId(dim.dimensionIndex)
+                context.lineWidth = 1.1 // hack--but fixes the 'disappearing lines' issue
```

Comment:
Not a huge fan of "magic numbers" like this, but it fixes the vanishing-line issue, at least for the example we have been looking at.